Assets/Scripts/Network/ColyseusClient.cs

```diff
@@ -60,13 +56,19 @@ private void Update()
         }
     }
 
-    public async Task ConnectToServer(string serverEndpoint, string username)
+    public async Task ConnectToServer(string serverEndpoint, string username, string roomName)
     {
         networkController = GetComponent<NetworkController>();
+        Debug.Log($"Connection: {!connecting}");
+        Debug.Log($"Connection: {!IsConnected}");
+
         if (!connecting && !IsConnected)
         {
+            Debug.Log("HERE WE ARE 1" + serverEndpoint);

```

Comment:
Should probably take these out before merging - our debug log is quite verbose as it is!